services/rollout-service/pkg/versions/versions.go

```diff
@@ -281,8 +283,13 @@ func (v *versionClient) ConsumeEvents(ctx context.Context, processor VersionEven
 
 			}
 
-			l.Info("version.push")
-			v.ArgoProcessor.Push(ctx, &overview)
+			overview.AppDetails = appsToChange
+			if err := v.ArgoProcessor.Push(ctx, &overview); err != nil {
+				l.Sugar().Warnf("version.push failed: %v", err)
+			} else {
+				l.Info("version.push")
```

Comment:
Is this log needed?

Reply:
This log is essential telling us when there is a push and if it fails. Since we now have a retry strategy it is not an error when a push fails, but I think we should still be able to see it in the logs that pushes have been failing so that we now why changes are not being applied to argocd